rootfs/etc/redis/redis.conf

```diff
@@ -0,0 +1,15 @@
+# The working directory.
+#
+# The DB will be written inside this directory, with the filename specified
+# above using the 'dbfilename' configuration directive.
+#
+# The Append Only File will also be created inside this directory.
+#
+# Note that you must specify a directory here, not a file name.
+dir /home/lsst
```

Comment:
Maybe use /tmp or something?  I dunno.  I guess this part is temporary, since this will eventually have to be broken out in its own service later (right now, if we restarted the container, since we don't mount home, we'd lose the redis db)

Reply:
Until we configure out the redis persistence question for the long term, /home/lsst is no worse than /tmp, for now.